tests/e2e/manifests/verify_manifest_lists.sh

```diff
@@ -26,20 +29,11 @@ fi
 LPATH=`dirname "$0"`
 cd "$LPATH"
 
-# this block acts like a cheap replacement for `godep` or `go get`.
-# we only need a sub-package so no need to pull the whole apimachinery tree.
-VERSION_PATH="src/k8s.io/apimachinery/pkg/util/version"
-mkdir -p $VERSION_PATH
-pushd $VERSION_PATH
-curl -sS "https://raw.githubusercontent.com/kubernetes/apimachinery/master/pkg/util/version/version_test.go" > "version_test.go"
-curl -sS "https://raw.githubusercontent.com/kubernetes/apimachinery/master/pkg/util/version/version.go" > "version.go"
-popd
-
-# set GOPATH to the local directory
-export GOMODULE=on
+# use go modules. this forces using the latest k8s.io/apimachinery package.
+export GO111MODULE=on
+go mod init
```

Comment:
Ideally we should have a go.mod file and pin to a version, but being the dependency super simple I guess we are fine also in this way

Reply:
the non-pining was intentional, because this also allows us to catch changes in the apimachinery version functions.

arguably not needed, but if this starts breaking constantly we can switch to a fixed go.mod.

Reply:
OK, thanks for the clarification 
Might be worth adding a comment in the code to keep track of the underlying reason of this at the first opportunity